flex/paths.py

```diff
@@ -133,20 +133,21 @@ def match_path_to_api_path(path_definitions, target_path, base_path='', context=
     assert isinstance(context, collections.Mapping)
```

Comment:
I ended up running with removing the `basePath` for the test schema -- let me know what you think!